### ISSUE TITLE
Getting GPU work when GPU suspended

### ISSUE THREAD

**Issue by hucker75** [BOINC/boinc#4623]
I suspended the GPU (to save power), while there was no queue and all projects set to no new work.  Then I set a few projects to allow new work.  It downloaded GPU work!  The GPU is suspended!  I now have a batch of WUs which will just sit there until they expire.

 - OS: Windows 11 (x64)
 - BOINC Version: 7.16.20 (x64)

**Comment by AenBleidd** [comment]
@hucker75, you should disable GPU tasks on Projects' page, local client doesn't look at the state of GPU because it doesn't know for how long it will be disabled, so it asks for the tasks according to the Projects' settings.

**Comment by hucker75** [comment]
Yes I've done that as a workaround.  But surely it shouldn't download until it's set to run again?  It could be off for a day or more.  And far more convenient for the user to be able to stop the GPU running with one click instead of going to possibly several project pages to turn it off.

**Comment by AenBleidd** [comment]
@hucker75, usually people either use GPU always (or according to their preferences) or not use it at all.

**Comment by hucker75** [comment]
Ok, maybe I'm unusual, but it does seem an illogical way of doing things.  Stick it on a "do sometime" list if you like.
